channelserver/README.md

```diff
@@ -0,0 +1,16 @@
+# Simple websocket pairsona server
+
+Config options:
+
+Options can be set as ENV vars (prefixed with "PAIR_", e.g.
+"PAIR_PORT="8000"), or as arguments ("--port=8000").
+
+See src/settings.rs for defaults.
+
+## Compile and run:
+
+After installing rust via rustup:
+
+$ cargo run
+
```

Comment:
Please add a description of the API offered by the server, and the messages that can be exchanged.